### PR TITLE
Add IManagementClient.GetShovelStatusesAsync() (api/shovels)

### PR DESCRIPTION
**Pull request by inikulshin**

### CODE REVIEW

File: Source/EasyNetQ.Management.Client/Serialization/DateTimeConverter.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace EasyNetQ.Management.Client.Serialization;
+
+internal class DateTimeConverter : JsonConverter<DateTime>
+{
+    public override DateTime Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+    {
+        Debug.Assert(typeToConvert == typeof(DateTime));
```
Comment (Pliner) [inline Source/EasyNetQ.Management.Client/Serialization/DateTimeConverter.cs:11]: Do we really need this here?

Comment (inikulshin) [reply Source/EasyNetQ.Management.Client/Serialization/DateTimeConverter.cs:11]: No, if we make `ShovelStatus.Timestamp` `string` rather than `DateTime`.

**Comment by inikulshin** [comment]
@Pliner what about the `GetShovelStatuses` name?

I thought that `GetShovels` (corresponding to `/api/shovels/`) will be confusing.
We have `CreateShovel`/`DeleteShovel` extension for creating/deleting shovel **parameter**, so `GetShovels` is supposed to be extension to `GetParameters`

I took `ShovelStatus` from https://www.rabbitmq.com/shovel-dynamic.html#status and used plural form `Statuses` because probably there will be a `GetShovelStatus`, returning single record.

**Comment by inikulshin** [comment]
@Pliner can you approve, please?
We need this change to finally move to latest EasyNetQ version from our custom build.

**Review by zidad — APPROVED** [review]

**Pull request merged**